raiden-ts/src/raiden.ts

```diff
@@ -1474,20 +1474,31 @@ export class Raiden {
             mergeMap(async (response) => response.text()),
             map((text) => {
               const suggestions = decode(SuggestedPartners, jsonParse(text));
-              assert(suggestions.length, ['empty partner suggestion from PFS', { pfs: pfs.url }]);
+
+              if (!firstResponse && !suggestions.length) {
+                firstResponse = [];
+                this.log.info('PFS returned no suggested partners', pfs);
```

Comment:
This log call will make it get logged twice: one here, and one inside the `catchError`

Reply:
Added it to the error.